keyboards/yosino58/info.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
This file only defines 42 keys, but `keyboards/yosino58/rev1/rev1.h` defines 58. These numbers need to match for QMK Configurator's support to work.

https://github.com/qmk/qmk_firmware/blob/0d110d8b5beeb1eaf4281212ed1c0c3dbd588968/keyboards/yosino58/rev1/rev1.h#L22-L28

Reply:
This is resolved.

:+1: